iOS/FVM/FVM.xcodeproj/project.pbxproj

```diff
@@ -427,7 +431,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.lapwingg.FVM;
+				PRODUCT_BUNDLE_IDENTIFIER = com.pega.FVM;
```

Comment:
Since Pega does not have any claims to this piece of software (correct me if I'm wrong), I'd change this to something like com.FVM.VehicleDamageModelling or similar; but that's just cosmetic, we can never mind it as well, up to you